docs/connectors/mysql.asciidoc

```diff
@@ -895,7 +895,52 @@ Other data type mappings are described in the following sections.
 [[temporal-values]]
 ==== Temporal values
 
-Other than MySQL's `TIMESTAMP` data type, the other MySQL temporal types depend on the value of the `time.precision.mode` configuration property. When `time.precision.mode` configuration property is set to `adaptive` (the default), then the connector will determine the literal type and semantic type for the temporal types based on the column's data type definition so that events _exactly_ represents the values in the database:
+Other than MySQL's `TIMESTAMP` data type, the other MySQL temporal types depend on the value of the `time.precision.mode` configuration property.
+
+[NOTE]
+====
+As of Debezium 0.7 `adaptive_time_microseconds` mode was introduced and is the default `time.precision.mode` for the MySQL connector. Mode `adaptive` was marked as deprecated.
+====
+
+[WARNING]
+====
+Debezium can only capture positive TIME field values in the range of `00:00:00.000000` to `23:59:59.999999` when `time.precision.mode` is set to `adaptive`, and `00:00:00.000` to `23:59:59.999` when `time.precision.mode` is set to `connect`.
+
+MySQL TIME fields value range is `-838:59:59.000000` to `838:59:59.000000`. This range can only be captured when `time.precision.mode` is set to `adaptive_time_microseconds`, which is available as of Debezium 0.7 .
```

Comment:
Superfluous whitespace before the last ".". Also I'd add a sentence:

> The `adaptive` and `connect` time precision modes should only be used if you can make sure that the  `TIME` values in your tables will never exceed the supported ranges. These modes will be removed in a future version of Debezium.

Reply:
s/todo/done/